muse_payment_adaptor/serializer.py

```diff
@@ -0,0 +1,148 @@
+from importlib.metadata import requires
+from muse_payment_adaptor.models import PaymentRequest, PaymentRequestDetails, PaymentResponse, PaymentResponseDetails
+from rest_framework import serializers
+
+
+class MessageHeader(serializers.Serializer):
+    msgId = serializers.CharField(max_length=50, source="message_id")
+    messageType = serializers.CharField(max_length=50)
+    paymentType = serializers.CharField(max_length=50, source="payment_type")
+
+
+class PaymentSummary(serializers.Serializer):
+    referenceNo = serializers.CharField(max_length=50, source="reference_no")
+    paymentDesc = serializers.CharField(
+        max_length=300, source="payment_description")
+    totalAmount = serializers.DecimalField(
+        max_digits=18, decimal_places=4, source="total_amount")
+    noofTransaction = serializers.IntegerField(source="no_of_tnx")
+    applyDate = serializers.DateTimeField(source="payment_date")
+
+
+class PaymentRequestDetailsSerializer(serializers.Serializer):
+    payeeCode = serializers.CharField(max_length=10, source="payee_code")
+    payeeName = serializers.CharField(max_length=50, source="name")
+    payeeAccountNumber = serializers.CharField(max_length=20, source="acc_no")
+    payeeAccountName = serializers.CharField(max_length=100, source="acc_name")
+    payeeBankName = serializers.CharField(max_length=50, source="bank")
```

Comment:
Can you be more explicit in DB field name. Change this from 'bank' to 'bank_name".